stingray/crossspectrum.py

```diff
@@ -240,7 +240,15 @@ def rebin(self, df, method="mean"):
         bin_cs.n = self.n
         bin_cs.norm = self.norm
         bin_cs.nphots1 = self.nphots1
-        bin_cs.nphots2 = self.nphots2
+        try:
+            bin_cs.nphots2 = self.nphots2
+        except AttributeError:
```

Comment:
You are not testing for this AttributeError, and that's probably why coverage decreased.